public/app/features/transformers/docs/content.ts

```diff
@@ -600,41 +625,55 @@ export const transformationDocsContent: TransformationDocsContentType = {
   | server 1                | 82   |          |
   | server 2                | 88.6 |          |
   | server 3                |      | 59.6     |
+
+  Utilize this transformation to construct a matrix by specifying fields from your query results. The matrix output reflects the relationships between the unique values in these fields. This helps you present complex relationships in a clear and structured matrix format.
```

Comment:
Always go with "use" rather than "utilize"
```suggestion
  Utilize this transformation to construct a matrix by specifying fields from your query results. The matrix output reflects the relationships between the unique values in these fields. This helps you present complex relationships in a clear and structured matrix format.
```